### DISCUSSION TITLE
Show Achievements tab in profile

### DISCUSSION THREAD

**Discussion by mmdbalkhi** [community/community#18246] [Other Feature Feedback, Questions, & Ideas]
hello, In my opinion, if it is in the tab profile with this name, it might be more interesting, I mean the following image:

![image](https://user-images.githubusercontent.com/65954744/173040001-77258c62-adfb-4bd1-99bd-be47f952b90a.png)

Of course, I think this will be done in the future because the URL has`tab` parameter Query(github.com/user **?tab=achievements** )

**Comment by MrKrishnaAgarwal** [comment]
Yes,
I can also see mine
https://github.com/MrKrishnaAgarwal?tab=achievements

**Comment by Faris0520** [comment]
hmm, i think just wait for github add more achievements. 

Because now there are only 3 achievements available, it's useless to create tabs that just contain small images. 🙏

**Reply by mmdbalkhi** [reply]
yep, and for now, this is beta

**Reply by Faris0520** [reply]
yeah 👍

**Reply by MrKrishnaAgarwal** [reply]
Let's hope for the best.
